### DISCUSSION TITLE
Show only one article or print only one article

### DISCUSSION THREAD

**Discussion by rfog** [FreshRSS/FreshRSS#5958] [Q&A] [answered]
I'm trying to use FreshRSS as a web sanitizer. I've got what I want enabling the custom CSS extension and now the view of each article is as I want. My next step is getting a PDF version of it, printing the article (as PDF), or saving it as PDF. However, at the bottom of the article, I have the next unread ones, like in the capture:

<img width="1159" alt="image" src="https://github.com/FreshRSS/FreshRSS/assets/1958131/6c1485f8-dd85-470d-b518-a6095480427d">

If I print that, I get at the end of the PDF the list of unread articles, expanded. Then I would like to be able to show only one article in the listing or not show the unread ones below the current one or have a "Print" button somewhere that will send only the open article to the printer. Is there any way to accomplish this?

**Comment by Alkarex** [comment]
In the search field, if you type `e:1703165842117260` adapting with the article ID (as visible e.g. when you hover the read or favourite icon), it will show only one article. We do not have a link for that at the moment. Does that help?

**Comment by rfog** [comment]
Wow! Very fast answer. THanks. 

This is the id:
<img width="311" alt="Screenshot 2023-12-21 150846" src="https://github.com/FreshRSS/FreshRSS/assets/1958131/dbdb5a1d-d60f-4ddb-a00d-9ffce5ad14aa">
And this what I type:
<img width="248" alt="image" src="https://github.com/FreshRSS/FreshRSS/assets/1958131/b9971f19-a887-4e59-94b6-5947118ee19c">
The result end with "There are no articles to show".

However, even if that will work, it is too complex to hover, hand write the ID, then go, and type the same long id without any error. My current version is 1.22.2-dev, as I want the custom CSS option and it does not work in current production.

Would it be interesting to have a feature request to add a print article button?

**Reply by Frenzie** [reply, marked as answer]
Could be we mostly just want some print styles akin to the custom CSS you mentioned? It could hide anything that isn't .active or whatever classname we use exactly.

**Comment by rfog** [comment]
Yes, that does the job:

`@media print {
    .flux.not_read {
        display: none;
    }
}`

Thanks.

(I will post here the custom CSS I'm using when I have it more polished).

**Comment by rfog** [comment]
Now I have another problem that I don't know where it comes from. On print, each image or link shows the link as text between brackets []. Is there any way to disable that? 

<img width="535" alt="image" src="https://github.com/FreshRSS/FreshRSS/assets/1958131/2c1e8f9b-2a00-4f56-aea4-d3884950acc7">

**Reply by Alkarex** [reply]
Please open a new issue or question, if it is not related to the parent one :-)

**Reply by rfog** [reply]
Sorry. Done.
